### ISSUE TITLE
CyanogenMod Crash

### ISSUE THREAD

**Issue by pihug12** [00-Evan/shattered-pixel-dungeon#234]
Hello!

Last version (0.6.3a) makes some CyanogenMod versions crash.  
Here is the bug: https://issuetracker.google.com/issues/64434571

Updating to Build Tools 26.0.3 should probably fix it.
https://github.com/00-Evan/shattered-pixel-dungeon/blob/master/core/build.gradle#L5

Thanks!

**Comment by 00-Evan** [comment]
Thanks for making me aware of this. I had researched this before but did not find any discussions about this problem, infact I made my own issue on google's issue tracker because it seemed like nobody else was aware of it: https://issuetracker.google.com/issues/69161604

I didn't update to build tools 26.0.3 initially because there was no associated release note, and I don't like updating dependencies without knowing what has changed. Clearly in this case updating build tools is important though. I was just about to publicly release 0.6.3b, so i'll add this change into that release.
